datafusion/core/src/datasource/physical_plan/parquet/reader.rs

```diff
@@ -112,10 +113,12 @@ impl AsyncFileReader for ParquetFileReader {
         self.inner.get_byte_ranges(ranges)
     }
 
-    fn get_metadata(
-        &mut self,
-    ) -> BoxFuture<'_, parquet::errors::Result<Arc<ParquetMetaData>>> {
-        self.inner.get_metadata()
+    fn get_metadata<'a>(
+        &'a mut self,
+        file_decryption_properties: Option<
+            &'a FileDecryptionProperties>
+    ) -> BoxFuture<'a, parquet::errors::Result<Arc<ParquetMetaData>>> {
+        self.inner.get_metadata(file_decryption_properties)
     }
```

Comment:
This more complex lifetime came from the new load_async stuff for metadata but may be wrong.